src/Dfe.PlanTech.Domain/Questionnaire/Models/AnswerDbEntity.cs

```diff
@@ -9,15 +11,20 @@ public class AnswerDbEntity : ContentComponentDbEntity, IAnswer<QuestionDbEntity
 
     public string? NextQuestionId { get; set; }
 
+    [DontCopyValue]
     public QuestionDbEntity? NextQuestion { get; set; }
 
     public string Maturity { get; set; } = null!;
 
     [DontCopyValue]
     public string? ParentQuestionId { get; set; }
 
+    [DontCopyValue]
```

Comment:
Just wanna check my understanding of this - am I right in thinking that the `AllowMultiple=true` is what allows you to put this on multiple attributes, and that by itself this doesn't change anything
_but_ the azure function is using the presence of DontCopyValue to ignore this attribute:
- in validation
- when incoming updates existing (because incoming => existing updates NextQuestionId and therefore there is no need to try and update NextQuestion because EF will do that.... and it would error if you tried to? or there another reason?)

Reply:
so this would need to be applied to every occurence of an `x` that comes from `xId` ?

Reply:
yeah it works so all for leaving it be 😄 thanks for explaining